app/lib/view/settings/subsettings/theme_changer.dart

```diff
@@ -1,8 +1,7 @@
 import 'package:flutter/material.dart';
```

Comment:
The theme changer is the wrong place to put this setting. This page is ment to be a "colors" like section and not a UI-elements section.
A page dedicated for the different applet settings would be better. That is something to consider adding with the implementation of #322

Reply:
This is clear, but there is no other settings component/page, and adding a new settings section is beyond the scope of this PR. Therefore, this is a temporary setting in place. Another option would be to wait until such settings are added or temporarily disable the option to disable it.

Reply:
I could add such page.